### ISSUE TITLE
Token Logits

### ISSUE THREAD

**Issue by Zoonjian** [OpenSQZ/MiniCPM-V-CookBook#96]
请问是否可以获得每个生成token的logits？model.chat接口似乎不支持
